sigmf-spec.md

```diff
@@ -663,22 +663,42 @@ this field be capable of displaying up to 20 characters.
 ### Collection Format
 
 The `sigmf-collection` file contains metadata in a single top-level object 
-called a `collection`. The Collection Object contains key/value pairs that 
-describe relationships between SigMF Recordings.
-
-The Collection object points to specific Recordings via a `SigMF Recording tuple`,
-which references the base-name of the Recording and the SHA512 hash of the 
-Metadata file. Tuples MAY be the singular value in a key/value pair, or provided
-in an ordered list via a JSON array.
+called a `collection`. The Collection object associates specific Recordings
+together through key/value pairs which link metadata keys and
+`SigMF Recording Objects` that point to Recordings containing actual data. The
+following rules apply to SigMF Collections:
 
 1. The Collection Object MUST be the only top-level object in the file.
-1. The Collection Object MUST only contain SigMF key/value pairs.
-1. Keys in the Collection Object MUST use SigMF Recording Tuples to reference 
+1. The Collection Object MUST only contain SigMF key/value pairs provided by the
```

Comment:
I would remove this rule from this list, and rather add it to the rules about Compliance later in the spec.

Reply:
Don't disagree....but this suggestion feels unrelated to this PR.